### DISCUSSION TITLE
WebDav sync of attached images

### DISCUSSION THREAD

**Discussion by adrien-delhorme** [damionx7/Zettel-Notes-Documentation#153] [General]
Hi,

I try to sync notes with Webdav (NextCloud) but I can't get attached images (stored in a `.attachments/` folder) to be synced.

Images on the server are not downloaded into the app, and images added in the app are not uploaded to the server.

Am I missing something in my setup?

**Comment by damionx7** [comment]
Hey, Dot folders are skipped from synchronization. Can you try after removing dot from attachments.

**Reply by adrien-delhorme** [reply]
:star_struck: it works! Thank you.
Is this behaviour enforced by the Webdav protocol or it is a choice?
Anyways I can submit a pull request to mention it in your doc, if you are interested.

**Reply by damionx7** [reply]
As dot folders are usually meant to be hidden, so they are excluded from synchronization. Please feel free to contribute.
